feature/stamps/src/main/java/com/momomomo111/stamps/StampsScreen.kt

```diff
@@ -0,0 +1,83 @@
+package com.momomomo111.stamps
```

Comment:
Sorry for the sudden message 🙌 
I believe it would be better to change this package to `io.github.droidkaigi.confsched2023.stamps` like a [`about feature`](https://github.com/DroidKaigi/conference-app-2023/blob/main/feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/AboutScreen.kt#L1).

Reply:
@momomomo111 Could you fix it? 😭

Reply:
Sorry, the revised PR is here.🙏🙏🙏

https://github.com/DroidKaigi/conference-app-2023/pull/540